rpc/wrtc_signaling_server.go

```diff
@@ -263,6 +250,8 @@ func (srv *WebRTCSignalingServer) Call(req *webrtcpb.CallRequest, server webrtcp
 				},
 			},
 		}); err != nil {
+			// Don't set an error for the connection attempt. Some candidates may have been
+			// exchanged that can result in a successful connection.
```

Comment:
Thanks as always for the in-line comments; totally agree with not sending error here.